src/block/block_compact.c

```diff
@@ -72,9 +88,9 @@ __wt_block_compact_progress(WT_SESSION_IMPL *session, WT_BLOCK *block, u_int *ms
         ++*msg_countp;
         __wt_verbose_debug(session, WT_VERB_COMPACT_PROGRESS,
           " compacting %s for %" PRIu64 " seconds; reviewed %" PRIu64 " pages, skipped %" PRIu64
-          " pages, cache pages evicted %" PRIu64 ", on-disk pages moved %" PRIu64,
+          " pages, pages rewritten %" PRIu64,
```

Comment:
Can we change it to rewritten %" PRIu64" pages ?

Reply:
Fixed in dc4cf63.